src/CoreLib/deploy.fs

```diff
@@ -217,10 +219,10 @@ type DeploymentSettings() =
     /// ReSerialization For Cache
     static member val internal DefaultCacheSerializationLimit = 1 with get, set
     /// ReSerialization For Cache
-    static member val internal DefaultIOMaxQueue = 10 with get, set
+    static member val internal DefaultIOMaxQueue = 100 with get, set
```

Comment:
Is this a good default for general use or it's optimized towards sort benchmark?